src/main/java/Duke.java

```diff
@@ -6,5 +8,58 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello Hello! I'm Duke\n" + "What can I do for you?");
+        int i = 0;
+        String txt;
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+
+        while (true) {
+            txt = in.nextLine();
+            if (txt.equals("list")){
+                System.out.println("Here are the tasks in your list:");
+                for(int x = 0; x < i; x++){
+                    System.out.printf("%d.%s\n", x+1, tasks[x]);
+                }
+            }else if (txt.startsWith("done ")){
+                int num;
+                num = Integer.parseInt(txt.substring(5));
+                tasks[num - 1].isDone = true;
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println(tasks[num - 1]);
+            }else if (txt.startsWith("todo ")){
```

Comment:
Should there be spaces between "}else" and ")){" to comply with coding standards?